lib/cli/run.ts

```diff
@@ -107,14 +111,16 @@ export const runHandler = async (args: CliRunOptions) => {
   });
 
   logger.print(`[OWL - CLI] Running tests on ${args.platform}.`);
-  await runProject(config, logger);
+
+  await runProject(config);
 
   const jestConfigPath = path.join(__dirname, '..', 'jest-config.json');
   const jestCommandArgs = [
     'jest',
     `--config=${jestConfigPath}`,
     `--roots=${cwd}`,
     '--runInBand',
+    `--globals='${JSON.stringify({ OWL_CLI_ARGS: args })}'`,
```

Comment:
Adding the Owl config as a global variable, so that it can be accessed from the `refresh` method during the test.

Reply:
I'm also considering adding a `__OWL__` or similar global here so that it can easily be used in the app to swap to using mock data etc during tests. 
A good use case would be to show a Story Book type screen during Owl tests only, so that all components can be easily tested.